admission-webhook/main.go

```diff
@@ -22,7 +22,12 @@ func main() {
 		panic(err)
 	}
 
-	webhook := newWebhookWithOptions(kubeClient, WithCertReload(*enableCertReload))
+	options := []WebhookOption{WithCertReload(*enableCertReload)}
+
+	randomHostname := env_bool("RANDOM_HOSTNAME")
+	options = append(options, WithRandomHostname(randomHostname))
+
+	webhook := newWebhookWithOptions(kubeClient, options...)
```

Comment:
Just for readability

```suggestion
	randomHostname := env_bool("RANDOM_HOSTNAME")
	
	options := []WebhookOption{WithCertReload(*enableCertReload)}
	options = append(options, WithRandomHostname(randomHostname))

	webhook := newWebhookWithOptions(kubeClient, options...)
```

Reply:
updated